circularSeekBar/src/main/java/me/tankery/lib/circularseekbar/CircularSeekBar.kt

```diff
@@ -1144,6 +1144,39 @@ class CircularSeekBar @JvmOverloads constructor(
         recalculateAll()
     }
 
+    private fun customPaint(paint: Paint, block: (Paint) -> Unit) {
+        block.invoke(paint)
+        invalidate()
+    }
+
+    fun customCircleFillPaint(block: (Paint) -> Unit) {
```

Comment:
Could you adding some Java docs for these public methods to explain what it does?